template/dockerfiles/gomodule/Dockerfile

```diff
@@ -5,7 +5,7 @@ EXPOSE {{PORT}}
 WORKDIR /go/src/app
 COPY . .
 
-RUN go mod vendor
+RUN go get
```

Comment:
Question for my understanding: why are we replacing go mod vendor with go get ?

Reply:
the vendoring isn't really necessary anymore, we just have it from project convention back when it was made, but go modules have improved significantly since then

[by default, go commands like go build ignore the vendor directory when in module mode.](https://go.dev/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away)